.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,16 @@
+Please answer these questions before submitting your issue.
```

Comment:
It might be better to say something like "Please answer these questions before submitting a bug report.", since these questions don't apply to feature requests.